lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -21,7 +21,15 @@
 import mayaUsd.ufe as mayaUsdUfe
 import mayaUsd.lib as mayaUsdLib
 import maya.internal.common.ufe_ae.template as ufeAeTemplate
-from maya.common.ui import LayoutManager
+
+try:
+    # This helper class was only added recently to Maya.
+    import maya.internal.ufeSupport.attributes as attributes
+    hasAEPopupMenu = 'AEPopupMenu' in dir(attributes)
+except:
+    hasAEPopupMenu = False
```

Comment:
Set a variable here which says whether or not we have the new AEPopupMenu class (from Maya).